.github/workflows/cypress-e2e-parallelization.yml

```diff
@@ -135,6 +144,95 @@ jobs:
           path: ${{ inputs.module_name }}/tests/e2e/cypress/results/reports/*.json
           retention-days: 1
 
+      - name: Upload xray reports
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ inputs.name }}-${{ inputs.os }}-xray-reports
+          path: ${{ inputs.module_name }}/tests/e2e/cypress/cucumber-logs/*.json
+          retention-days: 1
+
+  import-reports-to-xray:
```

Comment:
it is executed on every branches and PRs ?
Should not be executed only on release branches ?

Reply:
@tuntoja

Reply:
I think it should only be done on dev  + release branches, not on PRs
running it on PRs may generate a lot of changes/updates on xray, and since only dev branches lead to a potential release branches, it makes sense to run it on dev-xxxx / develop and release/hotfix branches only